safe-global/web-core
# chore: updated Dockerfile to be used in the safe-infrastructure project

## What it solves

Resolves safe-global/safe-infrastructure#58

## How this PR fixes it

Added new Dockerfile to be able to set environment variables [in the safe-infrastructure project](https://github.com/safe-global/safe-infrastructure)

## How to test it

To test the `web-core` image run:

- Make sure that you have a `.env` file defined
- Create the docker image: `docker build -t web-core .`
- Run the docker image: `docker run -p 3000:3000  web-core`
- Open your browser on `http://localhost:3000`
